doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3104,7 +3277,7 @@ NCSDK-23761: MCUboot fails to boot when both the :kconfig:option:`CONFIG_MCUBOOT
 
   **Workaround:** To fix the issue, disable either the :kconfig:option:`CONFIG_MCUBOOT_HW_DOWNGRADE_PREVENTION` or :kconfig:option:`CONFIG_BOOT_FIH_PROFILE_LOW` Kconfig option in MCUboot.
 
```

Comment:
@peknis One more item for MCUboot 

`KRKNWK-18776: MCUboot updates take a long time
  The current implementation of MCUboot does not optimally utilize RRAM, which causes longer update times.

  **Affected platforms:** nRF54L15 DK
`